packages/wdio-browserstack-service/src/Percy/PercySDK.ts

```diff
@@ -39,6 +73,27 @@ let screenshotAppHandler = async (...args: any[]) => {
     PercyLogger.error('Unsupported driver for percy')
 }
 if (percyAppScreenshot) {
-    screenshotAppHandler = percyAppScreenshot
+    screenshotAppHandler = (driverOrName: any, nameOrOptions?: any, options?: any) => {
```

Comment:
Can we properly type the parameters? Also, this function looks identifcal to `screenshotHandler` maybe it is wort to merge the code together to avoid duplicate code

Reply:
Typed the parameters & handled code duplication as well.